### ISSUE TITLE
Problems with signed gem instructions

### ISSUE THREAD

**Issue by mvz** [ledbettj/systemd-journal#63]
Hello,

There are two problems with your instructions for verifying the gem signature:

* The `wget` command downloads the html page containing the pem file, instead of just the pem file.
* The gem does not appear to actually be signed:
```
[matijs@bean] gem install systemd-journal -P HighSecurity
Fetching: systemd-journal-1.2.2.gem (100%)
ERROR:  While executing gem ... (Gem::Security::Exception)
    unsigned gems are not allowed by the High Security policy
```

**Comment by ledbettj** [comment]
Hi @mvz, thanks for reminding me -- I hadn't yet published a signed version.

I updated the url in the README and published signed version `1.2.3`.

**Comment by mvz** [comment]
Thanks, @ledbettj, it works now!
